src/index.html

```diff
@@ -1,13 +1,324 @@
 <!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Dia</title>
+    <title>Air</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;600&family=Poppins:wght@600&display=swap"
+      rel="stylesheet"
+      >
     <link rel="stylesheet" href="./styles/main.scss">
+    <script src="scripts/main.js" defer></script>
   </head>
+
   <body>
-    <h1>Dia</h1>
-    <script type="text/javascript" src="scripts/main.js"></script>
+    <header class="header" id="home">
+      <nav class="nav">
+        <div class="nav__container">
+          <a href="#home" class="logo nav__logo"></a>
+          <ul class="nav__list">
+            <li><a href="#about-us" class="nav__link">About us</a></li>
+            <li><a href="#services" class="nav__link">Services</a></li>
+            <li><a href="#work" class="nav__link">Work</a></li>
+            <li><a href="#contacts" class="nav__link">Contact Us</a></li>
+          </ul>
+
+          <div class="nav__hamburger">
+            <div class="nav__line"></div>
+            <div class="nav__line"></div>
+            <div class="nav__line"></div>
+          </div>
+
+          <div class="nav__full nav__nav__full--active">
+            <ul class="nav__list-full">
+              <li><a href="#about-us" class="nav__link">About us</a></li>
+              <li><a href="#services" class="nav__link">Services</a></li>
+              <li><a href="#work" class="nav__link">Work</a></li>
+              <li><a href="#contacts" class="nav__link">Contact Us</a></li>
+            </ul>
+          </div>
+
+          <a href="#contacts" class="nav__button">Hire us</a>
+        </div>
+      </nav>
+      <div class="header__container">
+        <div class="header__text">
+          <h1 class="header__title"> Strategic<br> Agency </h1>
+          <p class="header__description">
+             We believe in the power of bold ideas that
+             can solve business challenges.
+          </p>
+          <button class="header__button">Learn more</button>
+        </div>
+      </div>
+      <div class="slider">
+        <img
+          src="images/slider/slide-img-1.jpg"
+          alt="slide 1"
```

Comment:
```suggestion
          alt="Intro"
```
Better to give some meaningful values to `alt`